tests/unit/browser/test_adblock.py

```diff
@@ -114,14 +114,14 @@ def create_blocklist(directory, blocked_hosts=BLOCKLIST_HOSTS,
     return name
 
 
-def assert_urls(host_blocker, blocked=BLOCKLIST_HOSTS,
+def assert_urls(host_blocker, blocked=BLOCKLIST_HOSTS[1:],
```

Comment:
I think it'd be nicer to add `localhost` to the list of whitelisted URLs in this test.